### ISSUE TITLE
[bug] canary release - sidebar open/close

### ISSUE THREAD

**Issue by mickael-kerjean** [mickael-kerjean/filestash#770]
reported from https://github.com/mickael-kerjean/filestash/issues/629

```
Great work! Is the sidebar new? After clicking the sidebar icon (Your Files), the sidebar closes and there's no way to ever open it again.

EDIT: 14 inch laptop, chromium browser version 130.0.6723.69, computed browser width 1916px
```

**Comment by mickael-kerjean** [comment]
That should have been fixed in the new release

**Comment by ElderBlade** [comment]
I can confirm this is now fixed by [92673f9](https://github.com/mickael-kerjean/filestash/commit/92673f9ae3825e2c0bf449df4ca5a2abeaeb61af)
